be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -239,9 +241,11 @@ Status KafkaDataConsumer::group_consume(TimedBlockingQueue<RdKafka::Message*>* q
         case RdKafka::ERR_OFFSET_OUT_OF_RANGE: {
             done = true;
             std::stringstream ss;
-            ss << msg->errstr() << ", partition " << msg->partition() << " offset " << msg->offset() << " has no data";
+            ss << msg->errstr() << ", this is no data in partition: " << msg->partition()
```

Comment:
How about changing the error message to `offset out of range`, which is more common?

Reply:
users need to know kafka to understand `offset out of range`, this is specially modified before.